src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -28,7 +36,7 @@ void PreviewKeyDown(object sender, KeyEventArgs e)
                 RoutingStrategies.Tunnel);
         }
 
-        public static IDisposable Open(TopLevel root)
+        private static IDisposable Open(TopLevel root, DevToolsOptions options)
```

Comment:
Again, a breaking change. This method was public to allow showing DevTools immediately without pressing a key.